src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesTest.java

```diff
@@ -60,4 +60,40 @@ public void splitNamesIgnoringFullStopSpace_tabAfterFullStop_changeToSingleSpace
         assertThat(splitAroundAbbreviatedNames(nameWithTwoSpaces))
                 .isEqualTo(expected);
     }
+
+    @Test
+    public void splitNamesIgnoringApostropheSpace_ApostropheNoSpaceInName_doNotSplit() {
```

Comment:
Keep capitilisation consistent - the tests in here have lowerCase letters after every underscore. splitNamesIgnoringApostropheSpace_apostropheNoSpaceInName_doNotSplit not splitNamesIgnoringApostropheSpace_ApostropheNoSpaceInName_doNotSplit

Reply:
And as before, check the first part of the test name matches what is being tested.
splitAroundAbbreviatedNames_apostropheNoSpaceInName_doNotSplit

Reply:
Resolved.